src/App.js

```diff
@@ -1,7 +1,6 @@
-import './App.css';
-import Counter from "./components/Counter"
-import SearchInput from './components/SearchInput';
-import GenreList from './components/GenreList';
+import Counter from "./stories/Counter"
```

Comment:
I didn't get why it was decided to rename `components` to `stories. If it was done assuming that the storybook obliges us to do so, then it is not quite so.